cumulusci/tasks/metadata/package.py

```diff
@@ -249,6 +276,27 @@ def check_delete_excludes(self, item):
     def _parse_subitem(self, item, subitem):
         return [item + "/" + self.strip_extension(subitem)]
 
+    def _strip_component(self,item,component_list):
+        path = self.directory + "/" + item
```

Comment:
Let's use `os.path.join()` to avoid issues on Windows with the path separator.

The other option is to switch consistently to use Pathlib, but I'm okay with breaking that out into a later PR.